src/Couchbase.Lite.Support.NetDesktop/Activate.cs

```diff
@@ -75,7 +75,8 @@ public static void LoadLiteCore()
             }
 
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)) {
-				if(IntPtr.Size == 4) {
+                var arch = RuntimeInformation.OSArchitecture;
+                if (arch == Architecture.X86 || arch == Architecture.Arm) {

```

Comment:
Want to check if this is the correct way to get the arch from RuntimeInformation.OSArchitecture.

Reply:
I think this should be PlatformArchitecture, not OSArchitecture.

Reply:
There are only ProcessArchitecture and OSArchitecture.
ProcessArchitecture - The process architecture of the currently running app.
OSArchitecture - The platform architecture on which the current app is running. 
Ref: https://learn.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.runtimeinformation.osarchitecture?view=net-6.0

Reply:
Sorry ProcessArchitecture is what I mean.  From the docs it seems like ProcessArchitecture is what the current runtime is (i.e. it could be x86 runtime on x64 hardware) and OSArchitecture is the architecture of the OS (which would report x64 in the previous example)

Reply:
But actually I don't think there is anything wrong with the previous check.  On 32-bit systems, IntPtr size will be 4.

Reply:
I see. Since now we have x64 and arm64 process architectures, IntPtr size may not tell us additional info besides we know it's 64 bit?

Reply:
Technically I suppose it could be running on some unknown 64-bit system but I haven't yet heard of the .NET runtime on such a system so I put it in the category of "deal with it if it arises"